meetings/2022-10-19.md

```diff
@@ -0,0 +1,132 @@
+* SC: ??? . If we talk about read/write on a resource, we cannot expect to do that on a resource and expect the changes to be reflected on the container. 
+* TBL: It's a hack, but in use. 
+* HS: Doesn't have to be a hack. We can simply make this implicit behavior explicit. When a container is created, LDP teaches us to specify properties about that container in the POST, so we could use the same mechanism to specify that index.ttl had a special role... One would need to invent some vocabulary for that and agree to it.
+* eP: releavant old issue [Aligning representations of document and container resources with REST via single and compound state #198 ](https://github.com/solid/specification/issues/198)
+* SC: if the topic is changing behavior of the container, we have issues for that. 
+* KK: index.html issue is the most commented we have. Touches on many things (representation of a resource, client vs server responsibility, and so on). We should settle that discussion before 1.0. After that, we should not change this behavior. 
+* eP: +1 settle it before 1.0
+* SC: +1 we should start from use cases. we should not discourage use cases to make implementations easier. 
+* HS: Similar issue ???
+* SC: We can use input from LDP. In the LDP spec there are examples statements about non-containments being included in the container description. 
+* KK: LDP is part of the source of the problem because RDF Source is defined as a resource fully represented on RDF, non-RDF not represented in RDF, so it leaves out all of the resources which are *partially* represented in RDF??? of thing not represented on RDF. We are borrowing things from LDP, but it is not an LDP thing.
+* SC: Need to clarify what is in a container, and if a resource part of the same pattern is part of the container or not. 
+* SC: What KK mentions is not what we have been doing in Solid. 
+
+
+
+### HTTP Message Signatures RFC version 13
+
+* HS: The IETF is in last call for its HTTP Message Signatures RFC version 13 with feedback expected by the 25 October 2022: https://datatracker.ietf.org/doc/draft-ietf-httpbis-message-signatures/13/
+* HS: bblfish has an implementation in Scala compiling to JVM and JS: https://github.com/bblfish/httpSig/tree/trackIO
+* SC: Is https://github.com/bblfish/authentication-panel/blob/main/proposals/HttpSignature.md latest work?
+* TBL: I am interested to see mockups of login UX flows. Where do you expect people to store their keys... If browsers have enough key management support.
+* HS: I have this proposal ??? Solid Wallet Launcher App. Different panels for apps you use. Apps from the web that you don't know that much can ask for ???. Two options - one server-side, one client-side. Looking for people to design that. Currently have a thing on the back of a napkin :) 
```

Comment:
```suggestion
* HS: I have this proposal for a [Launcher App](https://github.com/bblfish/LauncherApp), which is now also en EU project [Solid Wallet](https://nlnet.nl/project/SolidWallet/index.html). It works like this: Secure JS is downloaded from your POD so is trusted by Origin which then launches a Page of your preferred apps.  You get to see different panels for apps you use (see picture below). Apps from the web that you don't know that much can ask for an http signature (how: see [code on github](https://nlnet.nl/project/SolidWallet/index.html). This can work in two ways: 
   + one client-side as described in [authorization panel Launcher App page](https://github.com/solid/authorization-panel/blob/main/proposals/LauncherApp.md), 
   + the other is it can also be done via a Proxy on one's POD.  Looking for people to design the UI for it. Currently have a thing on the back of a napkin :) 
   ![back of the napkin image of Launcher App](https://user-images.githubusercontent.com/124506/66335877-3485bc00-e93c-11e9-9861-b19beefea18e.png)
```